### DISCUSSION TITLE
How do I add the little tags to ops to show up in right bottom corner like “dbt”, “sql”

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#8554] [Q&A] [answered]
Hi! Can somebody tell my how can I add this little tags to ops in right bottom corner like "dbt", "sql", "ipynb" in Dagit UI like in this photo? ;)
 <https://mobile.twitter.com/olivierdupuis/status/1410698532894433284/photo/1>.
I cannot find any information about this feature and I think it helps visualy organise ops in graph.

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1636057043449400?thread_ts=1636057043.449400&cid=C01U954MEER).*

**Comment by yuhan** [comment, marked as answer]
You can set it using `@op(tags={"kind": ...})` like:
```python
@op(tags={"kind": "custom_tag_1"})
def op1():
    return 1


@op(tags={"kind": "custom_tag_2"})
def op2(i):
    ...


@job
def my_job():
    op2(op1())
```
Then, you will get a DAG in Dagit like this:
<img width="1392" alt="Screen Shot 2022-06-21 at 9 44 59 PM" src="https://user-images.githubusercontent.com/4531914/174945414-a8f75342-390a-4f6b-be05-cec6393cb3f7.png">


We haven't documented this feature yet -- There is an open issue https://github.com/dagster-io/dagster/issues/3210 which discussed about potential improvements that we may revisit in the future.

**Comment by will-holley** [comment]
Was this deprecated at some point?

**Reply by sryza** [reply]
Not that I'm aware of. Mind filing a Github issue if it's no longer working?
